packages/bot-web-ui/src/pages/tutorials/faq-content/__tests__/faq-content.spec.tsx

```diff
@@ -48,18 +26,78 @@ describe('<FAQContent />', () => {
                 </DBotStoreProvider>
             </StoreProvider>
         );
-        render(<FAQContent faq_list={mockedFaqContent} />, {
-            wrapper,
-        });
     });
 
     it('should render FAQContent', () => {
-        expect(screen.getByTestId('id-faq__wrapper')).toBeInTheDocument();
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        expect(screen.getByTestId('dt_faq_wrapper')).toBeInTheDocument();
+    });
+
+    it('should show content after clicking on accordion', () => {
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        const accordion = screen.getByTestId('dt_accordion_test');
+        userEvent.click(accordion);
+        expect(accordion).toHaveTextContent('Deriv Bot is a web-based strategy builder for trading digital options.');
+    });
+
+    it('should show faq content in text size-s on desktop', () => {
+        mock_store.ui.is_desktop = true;
+
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        const faq = screen.getByText('What is Deriv Bot?');
+        expect(faq).toHaveStyle('--text-size: var(--text-size-s);');
     });
 
-    it('should show content', () => {
-        const acc = screen.getByTestId('id-accordion-test');
-        userEvent.click(acc);
-        expect(acc).toHaveTextContent('Mocked FAQ Title 1');
+    it('should show faq content in text size-xs on mobile', () => {
+        mock_store.ui.is_desktop = false;
+
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+        const faq = screen.getByText('What is Deriv Bot?');
+        expect(faq).toHaveStyle('--text-size: var(--text-size-xs);');
+    });
+
+    it('should call handleAccordionOpen and accordion item should open on dektop', () => {
+        mock_store.ui.is_desktop = true;
+        render(<FAQContent faq_list={faq_content} />, { wrapper });
+
+        const accordion = screen.getByTestId('dt_accordion_test');
+        expect(accordion).toBeInTheDocument();
+
+        const faq = screen.getByText('What is Deriv Bot?');
+        userEvent.click(faq);
+        jest.advanceTimersByTime(5);
+        expect(
+            screen.getByText(
+                "Deriv Bot is a web-based strategy builder for trading digital options. It’s a platform where you can build your own automated trading bot using drag-and-drop 'blocks'."
```

Comment:
Same for this. It can be taken from a variable.

Reply:
Done ✅